.github/workflows/issue-labeler.yaml

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+name: Issue Labeling Workflow
+on:
+  issues:
+    types: opened
+
+jobs:
+  label_issues_using_regex:
+    runs-on: ubuntu-latest
+    steps:
+      - name: RegEx-based Issue Labeler
+        uses: github/issue-labeler@v2.5
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          configuration-path: .github/actions/issue-labeler/regexes-for-issue-labels.yaml
```

Comment:
Since this is more a configuration file than an action you could move it directly under `.github/`

Reply:
Updated!

Reply:
Should be the same as the path. I meant directly `.github/regexes-for-issue-labels.yaml` or `.github/issue-labeler-config-regexes.yaml`

Reply:
updated